Share The Load/Services/PersistanceService.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  PersistanceService.swift
+//  Share The Load
+//
+//  Created by Christopher Hefferman on 13/01/2021.
+//
+
+import Foundation
+import RealmSwift
+
+protocol PersistanceServiceType {
+    var answeredIds: [Int] { get }
+    func save(ids: [Int])
+    func save(questions: [Question])
+    func getQuestions() -> [Question]
+}
+
+/// Massive temporary hack to get save logic working
+/// We'll need a UserService to always give us the same instance of a user rather than this global properties
+/// Global properties are very bad.
+var user = User()
```

Comment:
This is the global one I was referring to

Reply:
Same as above comment. Not using anymore.